website/docs/02-About/3-branding.mdx

```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
+
+## Logomark Usage
+### Clear Space and Sizing
+Both the primary and secondary logos should always appear in their entirety, surrounded by a minimum area of clear space, so that headlines, text or other visual elements do not encroach on the logo. The minimum clear space is shown using the height of the brand icon. Logos should be used at a minimum size of 60px or 30mm (w).
+
+<img src="/img/brand/clear-space-and-sizing.png"/>
+
+### Logomark Misuse
+
+Do's and Don'ts
+
+Take care to maintain the integrity of the logomark. The rules demonstrated here apply to all versions of logos.
+
+Do not:
+
+1. Do not remove any asset of the logo
+2. Do not distort or skew the logo
+3. Do not modify the composition or its proportions
+4. Do not outline the typography or icon
+5. Do not add drop shadow/filters
+6. Do not frame the logo
+7. Do not use the logo on a background colour that renders any part of logo illegible.
+8. Do not angle the logo 09. Do not use the logo in an unspecified colour
+
+<img src="/img/brand/logomark-misuse.png"/>
+
+## Graphic Elements
+A suite of brand graphics are the ore of the Moja Global visual system and collectively work to represent the pillars of the brand: collaboration, innovation, flexibility and accessibility.
+
+Shapes combine and overlap in various configurations, whilst a bright pop of 'collaboration green' is used to highlight where shapes overlay, signifying the power of collaboration.
+
+<img src="/img/brand/graphic-elements.png"/>
+
+### Graphic Gradients
+
+In application, graphics further suggest these values through the merging and collaborating of color through gradients.
+
+These graphics can be used on their own as symbolism or together to interact with other content.
+
+<img src="/img/brand/graphic-gradients.png"/>
+
+### Placement and usage
+
+In application, an 8-column grid should be used and graphics should be applied at a 2-column, 3-column, or 4-column scale.
+
+Graphics can be placed liberally to accommodate other content and can be used fully or partially on the page.
+
+<img src="/img/brand/placement-and-usage.png"/>
+
+
+Example application: outdoor marketing
+
+<img src="/img/brand/outdoor-marketing.png"/>
+
+## Typography
+
+### Typographic Families
+
+DM Sans is a low-contrast geometric sans serif design, chosen for it's legibility and approachable rounded forms.
+
+It is a free font commissioned by Google from Colophon, an international and award-winning type foundry based in London (UK) and Los Angeles (US)
+
+[Download DM Sans](https://fonts.google.com/specimen/DM+Sans?preview.text_type=custom)
+
+<img src="/img/brand/typographic-families.png"/>
+
+### Typographic Hierarchy
+
+Hierarchy of text is important in a design because it allows the reader to understand, at a glance, the order of importance of the information at hand.
+
+This page exemplifies the sizing and proportions of various styles of text.
+
+<img src="/img/brand/typographic-hierarchy.png"/>
```

Comment:
Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.